### ISSUE TITLE
The color contrast ratio of the focus indicator on "Back" button with respect to background is less than 3:1

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode#152047]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#VisualStudioCode-Win32-Apr2021;#Visual Studio Code Client;#A11yWCAG2.1;#A11ySev2;#WCAG1.4.11;#DesktopApp;#Win10;#ColorContrast;

### Environment Details:
Visual Studio Code
Operating System: Windows 10 Pro 21H1

### Steps to Reproduce:

1. Launch the application.
2. Download the  quickinput-sample extension here: https://github.com/microsoft/vscode-extension-samples/tree/main/quickinput-sample
3. F5 the sample extension.
4. Run "Quick Input Samples" from the command palette
5. Pick "multiStepInput" and then TAB to any option and select it using ENTER key.
6. The next step will have a "Back" button that demonstrates the original issue.
7. TAB to "Back" button.
8. Open AI4D color contrast analyzer and check the color contrast ratio of the focus indicator on "Back" button with respect to background

### Actual:

The color contrast ratio of the focus indicator on "Back" button with respect to background is 2.663:1 which is less than 3:1.

### Expected:
The color contrast ratio of the focus indicator on "Back" button with respect to background should be greater than or equal to 3:1.

### User Impact:
The low vision users won't be able to see the focus on the "Back" button properly.

**Comment by TylerLeonhardt** [comment]
After looking at this with @daviddossett, this seems to also apply to the main titlebar on macOS so we should come up with a generic solution for the quick pick titlebar and the main titlebar.

![image](https://user-images.githubusercontent.com/2644648/176555475-925ce24d-4767-446e-bfbd-2dd893ccd1cc.png)
![image](https://user-images.githubusercontent.com/2644648/176555496-68df0a2a-b461-48ac-89f8-a60987c76668.png)

@sbatten do you know of any other places where we may have this problem? Do you have any thoughts on the right direction to take here? 

David and I tried aligning the link active foreground and the focus hover but it didn't look great...

**Comment by daviddossett** [comment]
For more context, we have some precedent for using different focus borders with things like the status bar, which are already blue in our most common default themes. We use `statusBar.focusBorder` and `statusBarItem.focusBorder` for this.

<img width="708" alt="CleanShot 2022-06-30 at 08 30 09@2x" src="https://user-images.githubusercontent.com/25163139/176717055-2032cde2-5f3b-4eba-aafd-74a1c5eb7844.png">

Probably the least intrusive way to do this would be to have something like `titleBarItem.focusBorder` and reuse it for the quick pick as well since its somewhat coupled with the title bar. Especially with the new command center.

**Comment by TylerLeonhardt** [comment]
Yeah if it really is just those 2 locations we could introduce two keys?

**Comment by isidorn** [comment]
+1 for introducing a new key, or 2 keys.
Adding accessibility-sla label since we have to fix this one in the next 2 months.
